modules/test/playwright/tests/object-web/objectFields.spec.ts

```diff
@@ -94,15 +94,24 @@ test.describe('Manage object fields through Model Builder', () => {
 	}) => {
 		const {listTypeDefinitionIds, objectDefinition} = createdEntities;
 
-		const listTypeDefinitions = await Promise.all(
-			Array(22)
-				.fill(null)
-				.map(() =>
-					apiHelpers.listTypeAdmin.postRandomListTypeDefinition()
-				)
+		let allListTypeDefinitions = (
+			await apiHelpers.listTypeAdmin.getListTypeDefinitions()
+		).items;
+
+		allListTypeDefinitions = allListTypeDefinitions.concat(
```

Comment:
Hey @p-albuquerque, could you create two const `existingListTypeDefinitions` and `allListTypeDefinitions` to make the code more readable?

```ts
const existingListTypeDefinitions = (
	await apiHelpers.listTypeAdmin.getListTypeDefinitions()
).items;

const allListTypeDefinitions = existingListTypeDefinitions.concat(

...

```